docs/get-started/v2/get-started/features/recordings/storage.mdx

```diff
@@ -59,44 +65,54 @@ You can configure the **`type`** field of recording object to `s3` for AWS, `oss
 - Region: Name of the region where your bucket is located in
 - Prefix for Upload Path: Define the directory name (optional)
 
+</Tab>
+
 ## Fetching the asset
 
-### Using the 100ms Dashboard
+<Tabs id="fetch-recordings" items={['Dashboard','REST API','Webhooks']} />
+
+<Tab id='fetch-recordings-0'>
 
 You can access your recordings on the [sessions page](https://dashboard.100ms.live/sessions) in the 100ms Dashboard.
 
 ![Recording Links](/docs/v2/recording-links-session.png)
 
-### Using the REST API
+</Tab>
+
+<Tab id='fetch-recordings-1'>
 
 Recordings generate [recording assets](/server-side/v2/api-reference/recording-assets/overview) that can be fetched with the REST API.
 
-### Get webhooks
+</Tab>
+
+<Tab id='fetch-recordings-2'>
 
 100ms can send webhooks when the recording has stopped and is available for download. The recording path is available in following webhook responses:
 
-- Room composite (also called browser) recording: [beam.recording.success](/server-side/v2/introduction/webhook#beamrecordingsuccess) (attribute: `recording_path`)
-- Room composite - legacy (also called SFU) recording: [recording.success](/server-side/v2/introduction/webhook#sfu-recording-events) (attribute: `recording_path`)
-- Recordings enabled with live streaming (HLS): [hls.recording.success](/server-side/v2/introduction/webhook#hlsrecordingsuccess)
+- Room Composite (also called browser) recording: [beam.recording.success](/server-side/v2/introduction/webhook#beamrecordingsuccess) (attribute: `recording_path`)
+- Room Composite - Legacy (also called SFU) recording: [recording.success](/server-side/v2/introduction/webhook#sfu-recording-events) (attribute: `recording_path`)
+- Live Stream Recording (HLS): [hls.recording.success](/server-side/v2/introduction/webhook#hlsrecordingsuccess)
```

Comment:
```suggestion
- HLS Recording (VOD): [hls.recording.success](/server-side/v2/introduction/webhook#hlsrecordingsuccess)
```

Reply:
Calling it HLS Recording only. Removed (VOD)